portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,15 +1,18 @@
 package com.google.sps.data;
+import java.time.*; 
```

Comment:
This type of import is forbidden in our style guide. Imported values need to be explicit.

Reply:
Is there a specific reason for it being forbidden in addition to code clarity?

Reply:
I admire your will to question the status quo.

Our style guide is terse on this;
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports
```
Wildcard imports, static or otherwise, are not used.
```

We have a linter internally which flags these, statements. It links to the following explanation:

> They make code brittle and difficult to reason about, both for programmers and for tools. In the following example, the processing of the first import requires reasoning about the classes Outer and Nested, including their supertypes, and ends up depending on information in the second import statement. This code is (incorrectly) rejected by the latest version of javac. Also, note that I is actually being imported via the name p.q.C.I even though it's not declared in C! Its canonical name is p.q.D.I. Regular single-type imports require that all types are imported by their canonical name, but static imports do not.

---

FWIW, this is a very common preference in many languages (whether they include this functionality or not). Here's what the Rust community, a community I follow, says about this
https://rust-lang.github.io/rust-clippy/master/index.html#wildcard_imports

> wildcard imports can polute the namespace. This is especially bad if you try to import something through a wildcard, that already has been imported by name from a different source:
> 
> This can lead to confusing error messages at best and to unexpected behavior at worst.